server/portal/apps/datafiles/views.py

```diff
@@ -38,6 +38,15 @@ def get(self, request):
         default_system = user_systems[settings.PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEM_DEFAULT]
         response['default_host'] = default_system['host']
 
+        for system in response['system_list']:
+            try:
+                if system['api'] == 'tapis':
+                    system['definition'] = request.user.agave_oauth.client.systems.get(
+                        systemId=system['system']
+                    )
+            except Exception:
```

Comment:
I'm not getting any response on an Exception here, with a projects entry in the `PORTAL_DATAFILES_STORAGE_SYSTEMS` entry:
```

    {
        'name': 'Shared Workspaces',
        'scheme': 'projects',
        'api': 'tapis',
        'icon': None
    },
```